projects/hostap/libfuzzer_entry.c

```diff
@@ -0,0 +1,34 @@
+#include <errno.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+
+#ifndef EXTRA_ARGS
+#define EXTRA_ARGS
+#endif
+
+int fuzzer_main(int argc, char *argv[]);
+
+// Wrapper for AFL compatible fuzzer.
```

Comment:
This comment is not exactly correct. The function defined below is needed for libFuzzer fuzzing engine. For AFL, we're using [afl_driver ](https://github.com/llvm-mirror/compiler-rt/blob/master/lib/fuzzer/afl/afl_driver.cpp) that wraps around `LLVMFuzzerTestOneInput` and enables us to run AFL on the same targets.

Reply:
Done.